package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.11.2-beta.0",
+  "version": "0.11.3-dev.20240816092154",
```

Comment:
This seems good to go! Let's just wait for the internal CI/CD to be green and then we can revert this dev version, merge this and release the final version!